.rubocop.yml

```diff
@@ -46,7 +46,7 @@ Metrics/AbcSize:
   Max: 391
 
 Metrics/ClassLength:
-  Max: 235
```

Comment:
Instead of changing the rule, could you use the rubocop:disable comment to turn this rule off for that one class?
# rubocop:disable Metrics/ClassLength